src/bar-chart/bar.component.ts

```diff
@@ -37,7 +36,7 @@ import { DataItem } from '../models/chart-data.model';
 })
 export class BarComponent implements OnChanges {
   @Input() fill;
-  @Input() data: DataItem;
+  @Input() data: any;
```

Comment:
What happened here? I'm curious.

Reply:
Rollup complaining about DataItem not being exported. It is.